wrappers/python/demo/test.py

```diff
@@ -135,113 +128,14 @@
     }
 )
 
-rev_state = CredentialRevocationState.create(
-    rev_reg_def_pub,
-    revocation_status_list,
-    rev_idx,
-    rev_reg_def_pub.tails_location,
-)
-
-schemas = { schema_id: schema }
-cred_defs = { cred_def_id: cred_def_pub }
-rev_reg_defs = { rev_reg_id: rev_reg_def_pub }
-rev_status_lists = [issued_rev_status_list]
-
-present = PresentCredentials()
-
-present.add_attributes(
-    recv_cred,
-    "attr1_referent",
-    reveal=True,
-    timestamp=time_after_creating_cred,
-    rev_state=rev_state
-)
-
-present.add_attributes(
-    recv_cred,
-    "attr2_referent",
-    reveal=False,
-    timestamp=time_after_creating_cred,
-    rev_state=rev_state
-)
-
-present.add_attributes(
-    recv_cred,
-    "attr4_referent",
-    reveal=True,
-    timestamp=time_after_creating_cred,
-    rev_state=rev_state
-)
-
-present.add_predicates(
-    recv_cred,
-    "predicate1_referent",
-    timestamp=time_after_creating_cred,
-    rev_state=rev_state
-)
-
-presentation = Presentation.create(
-    pres_req,
-    present,
-    {"attr3_referent": "8-800-300"},
-    master_secret,
-    schemas,
-    cred_defs,
-)
-
-verified = presentation.verify(
-    pres_req,
-    schemas,
-    cred_defs,
-    rev_reg_defs,
-    rev_status_lists
-)
-assert verified
-
-# Issuer revokes credential
-
-time_revoke_cred = time_after_creating_cred + 1
-revoked_status_list = issued_rev_status_list.update(
-    time_revoke_cred,
-    None,
-    [rev_idx],
-    rev_reg_def_pub
-)
-
-rev_status_lists.append(revoked_status_list)
-
-rev_state.update(
-    rev_reg_def_pub,
-    revocation_status_list,
-    rev_idx,
-    rev_reg_def_pub.tails_location,
-    revoked_status_list,
-)
-
-present = PresentCredentials()
-present.add_attributes(
-    recv_cred,
-    "attr1_referent",
-    reveal=True,
-    timestamp=time_revoke_cred,
-    rev_state=rev_state
-)
-
-present.add_attributes(
-    recv_cred,
-    "attr2_referent",
-    reveal=False,
-    timestamp=time_revoke_cred,
-    rev_state=rev_state
-)
-
-present.add_attributes(
```

Comment:
Why removed?

Reply:
Ah I think an issue occurred with mergin. Will revert it.